src/main/resources/META-INF/plugin.xml

```diff
@@ -6,12 +6,21 @@
 
   <description><![CDATA[
       <img src="https://raw.githubusercontent.com/redhat-developer/intellij-openshift-connector/master/src/main/resources/images/openshift_extension.png" width="32" height="32"/>
-      <h2>Overview</h2>
       <p>A plugin for interacting with Red Hat OpenShift using a local or remote instance of OpenShift clusters providing a streamlined developer experience. This extension is currently in Preview Mode and supports only Java and Node.js components. We will be supporting other languages in the future releases.</p>
```

Comment:
It will be good to change the plugin description and match it with what we have on VSCode Marketplace. We also need to remove Preview mode.

Let's add these lines and remove the current one: 

The OpenShift Toolkit provides all of the power and convenience of IDEs for developing cloud-native Kubernetes applications, without leaving the comfort of your IDE. For a developer to run the local code on an OpenShift instance, test & debug it and deploy it on production without worrying about the complexities around different K8s tools.

OpenShift Toolkit extension provides an end-to-end developer experience for Red Hat® OpenShift®. Using this extension:

- Developers can easily create, deploy and debug container applications running on OpenShift.
- Create [devfile](https://devfile.io/) based components directly from Devfile Registries View
- Developers can view and edit Resources YAML manifests, and view logs for pods, deployments, and deployment configs.
- Provision free [Developer Sandbox for Red Hat OpenShift](https://developers.redhat.com/developer-sandbox) instance from IDE

Reply:
see also issue #483 for updating supported clusters